src/components/NcRichText/NcRichText.vue

```diff
@@ -410,6 +419,12 @@ export default {
 	},
 	emits: ['interact:todo'],
 
+	setup(props) {
+		if (props.useExtendedMarkdown && !remarkGfmLoaded.value) {
+			importRemarkGfmLibrary()
+		}
+	},
```

Comment:
This loads the lib only if `useExtendedMarkdown` was true initially. It doesn't support prop update.